README.md

```diff
@@ -1,24 +1,81 @@
-# README
+![](https://img.shields.io/badge/Ali-Atayee-yellow?labelColor=black)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# ALFA BUDGET App
 
-Things you may want to cover:
+> The Budget app is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Ruby version
+## Major Technologies
+- [ ] Ruby on Rails
+- [ ] Postgresql
+- [ ] TailwindCSS
 
-* System dependencies
+### [Video Presentation Link](https://www.loom.com/share/027dc24ffb754aab86db457fddc06530)
+### Install & Setup
 
```

Comment:
- [ ] I did not see a piece of code that is doing expense calculations either in the video or in the project. Please you can point me in the right direction to that code.